torch_ecg/api/config.py

```diff
@@ -0,0 +1,23 @@
+from pathlib import Path
+
+from pydantic_settings import BaseSettings
+
+
+class Settings(BaseSettings):
+    APP_HOST: str = "127.0.0.1"
+    APP_PORT: int = 8000
+    APP_WORKERS: int = 2
+
+    # Middleware settings
+    ENABLE_AUTHENTICATION: bool = True
+    BASIC_USERNAME: str = "username"  # Credentials to access the application
+    BASIC_PASSWORD: str = "password"
```

Comment:
I didn't see how the password are managed: it looks like it is defined in clear here? A question for the interview: which solutions to avoid commit passwords in the code?